goldens/public-api/angular_devkit/schematics/index.md

```diff
@@ -841,6 +842,8 @@ export function source(tree: Tree_2): Source;
 // @public (undocumented)
 const standardFormats: schema.SchemaFormat[];
 
+export { strings }
```

Comment:
Nit: Missing semicolon.

Reply:
That file is auto generated by the public api rule.